CMakeLists.txt

```
@@ -1454,12 +1454,27 @@ if(CONFIG_BUILD_OUTPUT_BIN)
 endif()
 
 if(CONFIG_BUILD_OUTPUT_BIN AND CONFIG_BUILD_OUTPUT_UF2)
+  if(CONFIG_BUILD_OUTPUT_UF2_IGNORE_FLASH_BASE)
+    set(FLASH_ADDR "0x0")
```

I don't believe `FLASH_ADDR` should be considered a globally used variable.

We try to indicate locally used vars by using lower case.
```suggestion
    set(flash_addr "0x0")
```

---

Fixed.